8.2.14 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.14

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

su-sws-jira committed:
Updated dependencies Jul 8 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

su-sws-jira committed:
Updated dependencies Jul 15 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

kaseybon committed:
Sparkbox Sprint 20 (#653)

- fix(STN-438): Sprint 19 QA Feedback (#646)
- fix: missing bracket in media/video-with-caption.js (#649)
- feat(STN-267): style the .more-link class as a secondary button (#651)
- feat(STN-398): .hb-raised-card styles for the Colorful theme (#650)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

pookmish committed:
HSD8-824 Allow paragraph field to be altered

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

pookmish committed:
HSD8-824 remove private paragraphs from basic page

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

pookmish committed:
HSD8-866 Fixed pagers with distinct settings

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs

pookmish committed:
HSD8-853 Update default config settings (#655)

* HSD8-853 Update default settings, permissions, and views

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr648-1jmy0v8eliomzgxtrmtwwrvqkhsqncap.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253

Visual Diffs:
- https://dashboard.tugboat.qa/5f05081441cd26315d5f4253/visualdiffs
